### ISSUE TITLE
CurseForge incorrect 'Requires Cloth API' image

### ISSUE THREAD

**Issue by supersaiyansubtlety** [Globox1997/VillagerQuests#7]
The image on the mod's CurseForge page states 'Requires Cloth API', but in reality Cloth *Config* API is required. 
[Cloth Config API on CurseForge](https://www.curseforge.com/minecraft/mc-mods/cloth-config)
[Cloth API on CurseForge](https://www.curseforge.com/minecraft/mc-mods/cloth-api)

**Comment by Globox1997** [comment]
Yes that is kinda misleading but if you click on the icon it will lead you to cloth config api.
Don't know how to create the badge otherwise
